package.json

```diff
@@ -53,23 +53,23 @@
     "Aptos SDK"
   ],
   "peerDependencies": {
-    "axios": "^1.8.0",
+    "axios": ">=1.8.2",
```

Comment:
I suggest we keep the ^

>= means the package version must be at least 1.8.2, but there is no upper limit, so it allows any newer version, including major version and can introduce breaking changes if a major version upgrade happens. 

While ^ allows updates only within the same major version, 
this ensures compatibility while allowing non-breaking updates.

Reply:
What if we do... >=1.8.2,<=1.9.0 ?

Reply:
Sure,or >= 1.8.0 <2.0.0
Both the same as ^